iothub_client/src/blob.c

```diff
@@ -224,12 +224,13 @@ BLOB_RESULT Blob_UploadMultipleBlocksFromSasUri(const char* SASURI, IOTHUB_CLIEN
                                     size_t size; /* source size set by getDataCallback */
 
                                     /* get first block */
-                                    getDataCallback(FILE_UPLOAD_OK, &source, &size, context);
+                                    IOTHUB_CLIENT_FILE_UPLOAD_GET_DATA_RESULT cbkRet;
```

Comment:
Sorry, we usually write more explicit var names... would you mind changing it something like "getDataReturnValue"?

Reply:
Fair enough, cbkRet is not very explicit.